documentation/jetty/modules/operations-guide/pages/start/start-jpms.adoc

```diff
@@ -73,24 +189,24 @@ However, when running on the module-path, things are quite different.
 When Jetty tries to load, for example, class `org.postgresql.ds.PGConnectionPoolDataSource`, it must be in a JPMS module that is resolved in the run-time module graph.
 Furthermore, any dependency, for example classes from the `java.sql` JPMS module, must also be in a module present in the resolved module graph.
 
-Thanks to the fact that when Jetty starts in JPMS mode the `--add-modules ALL_MODULE_PATH` option is added to the JVM command line, every `+*.jar+` file in the module-path is also present in the module graph.
+Thanks to the fact that when Jetty starts in JPMS mode the `--add-modules=ALL-MODULE-PATH` option is added to the JVM command line, every `+*.jar+` file in the module-path is also present in the module graph.
 
 There are now two cases for the `postgresql-<version>.jar` file: either it is a proper JPMS module, or it is an automatic JPMS module (either an explicit automatic JPMS module with the `Automatic-Module-Name` attribute in the manifest, or an implicit automatic JPMS module whose name is derived from the `+*.jar+` file name).
 
 If the `postgresql-<version>.jar` file is a proper JPMS module, then there is nothing more that you should do: the `postgresql-<version>.jar` file is in the module-path, and all the modules in the module-path are in the module graph, and any dependency declared in the `module-info.class` will be added to the module graph.
 
 Otherwise, `postgresql-<version>.jar` file is an automatic module, and will likely have a dependency on the JDK-bundled `java.sql` JPMS module.
-However, the `java.sql` JPMS module is not in the module graph, because automatic modules do not have a way to declare their dependencies.
+Fortunately, when Jetty starts in JPMS mode, the `--add-modules=ALL-DEFAULT` option is also added to the JVM command line, so that all the Java JPMS modules are added to the module graph, and this would include the `java.sql` JPMS module.
 
-For this reason, you have to manually add the `java.sql` dependency to the module graph.
-Using the `postgresql.mod` introduced in xref:start/index.adoc#configure-custom-module[this section] as an example, modify your custom module in the following way:
+In case you need to add a custom library JPMS module to the module graph, you have to manually add it to the module graph.
```

Comment:
This reads awkwardly.

Can it be ...

```suggestion
In case you need to add a custom library as a JPMS module to the module graph, you have to manually add it to the module graph via the normal JPMS means (module-info.class, command line, etc)
```

Reply:
Turned out the section was describing an old behavior that is now unnecessary, so I made more changes.